Apsitvarkom.Api/Program.cs

```diff
@@ -21,6 +21,7 @@
     options.AddPolicy(FrontEndPolicy, policy => policy.WithOrigins(builder.Configuration.GetValue<string>("FrontEndOrigin")));
 });
 
+builder.Services.AddScoped<IPollutedLocationContext>(provider => provider.GetService<PollutedLocationContext>()!);
```

Comment:
Why not `GetRequiredService` instead of `GetService` ?

Reply:
Good point. Didn't consider this at all. For any others that would read this and wouldn't be aware of the difference, more information [here](https://andrewlock.net/the-difference-between-getservice-and-getrquiredservice-in-asp-net-core/).

Reply:
I think you also don't need the `!` at the end then